cmd/configschema/docsgen/main.go

```diff
@@ -15,14 +15,13 @@
 package main
 
 import (
-	"go.opentelemetry.io/collector/service/defaultcomponents"
-
 	"github.com/open-telemetry/opentelemetry-collector-contrib/cmd/configschema"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/cmd/configschema/docsgen/docsgen"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/components"
 )
 
 func main() {
-	cmps, err := defaultcomponents.Components()
+	cmps, err := components.Components()
```

Comment:
This seems like a significant change since this is a much larger set of components.  What effect does this have?

Reply:
Thank you for pointing this out @Aneurysm9; this has been fixed!